code/modules/admin/verbs/secrets.dm

```diff
@@ -335,7 +335,7 @@
 				return
 			SSblackbox.record_feedback("nested tally", "admin_secrets_fun_used", 1, list("Egalitarian Station"))
 			for(var/obj/machinery/door/airlock/W in GLOB.machines)
-				if(is_station_level(W.z) && !istype(get_area(W), /area/bridge) && !istype(get_area(W), /area/crew_quarters) && !istype(get_area(W), /area/security/prison))
+				if(is_station_level(W.z) && !istype(get_area(W), /area/command) && !istype(get_area(W), /area/commons) && !istype(get_area(W), /area/service) && !istype(get_area(W), /area/command/heads_quarters) && !istype(get_area(W), /area/security/prison))
```

Comment:
Why the change here?

Reply:
This code targeted all of "crew quarters" for its things. Crew quarters, for some reason, put command staff quarters, common areas, and the service department all under one parent. To make sure it still targets all the same areas, I separated them into their respective new area paths.

Reply:
aight sounds good, just wanted to check